### PR TITLE
NimOS SDK package versioning

### PR DESCRIPTION
**Pull request by ghost**
Updated the testing logic to import the **nimbleclient** module and then use its location instead of relative paths from **tests** folder path.

Signed-off-by: Mandar Shivrekar <mandar.shivrekar@hpe.com>

### CODE REVIEW

**Comment by ghost** [comment]
testing:
```
$ pytest test_version.py 
================================================ test session starts =================================================
platform linux -- Python 3.6.8, pytest-5.4.2, py-1.8.1, pluggy-0.13.1 -- /usr/local/bin/python3.6
cachedir: .pytest_cache
rootdir: /auto/home.nas02/mshivrekar/git/nimble-python-sdk/tests, inifile: pytest.ini
collected 1 item                                                                                                     

test_version.py::test_nimos_sdk_package_versioning PASSED

-------- generated xml file: /auto/home.nas02/mshivrekar/git/nimble-python-sdk/tests/logs/pytest_summary.xml ---------
================================================= 1 passed in 2.19s ==================================================

$ cat logs/TestcaseRun.log 

 28-05-2020 20:17:41 **** Starting Tests for NimOS SDK package versioning TestCase *****

 28-05-2020 20:17:41 Obtained '1.0.0' version for NimOS SDK package
 28-05-2020 20:17:41 **** Completed Tests for NimOS SDK package versioning TestCase *****
```

File: tests/test_version.py [hunk 1]
```diff
@@ -4,17 +4,19 @@
 
 import pytest
 import os
```
Comment (ar-india) [inline tests/test_version.py:6]: we don't need import os

**Review by ar-india — APPROVED** [review]
Please remove "import os" 
rest all looks good to me

File: tests/test_version.py [hunk 2]
```diff
@@ -4,17 +4,19 @@
...
+import nimbleclient
+from tests.nimbleclientbase import SKIPTEST, log_to_file as log
 
 '''Version check test for NimOS package'''
```
Comment (ar-india) [inline tests/test_version.py:10]: this should be 
Version check for Nimble Python sdk

File: tests/test_version.py [hunk 3]
```diff
@@ -4,17 +4,19 @@
 
 import pytest
 import os
+import nimbleclient
```
Comment (ar-india) [inline tests/test_version.py:7]: although this won't create a problem. but can we replace this as below
`import nimbleclient `=> imports everything inside nimbleclient
`from nimbleclient import __init__ ` ==> just imports init

Comment (ar-india) [reply tests/test_version.py:7]: note that you will also need to modify 
` with open(nimbleclient.__file__) as fh:`  and replace with  `with open(__init__.__file__) as fh:`

File: tests/test_version.py [hunk 4]
```diff
@@ -4,17 +4,19 @@
...
 def test_nimos_sdk_package_versioning():
     log("**** Starting Tests for NimOS SDK package versioning TestCase *****\n")
 
     version = None
-    base_path = os.path.dirname(__file__)
-    with open(os.path.join(base_path, '..', 'nimbleclient', '__init__.py')) as fh:
+    with open(nimbleclient.__file__) as fh:
         info = fh.read()
         version = [line for line in info.split('\n') if line.startswith('__version__')][0].split(' = ')[1][1:-1]
         log("Obtained {!r} version for NimOS SDK package".format(version))
```
Comment (ar-india) [inline tests/test_version.py:22]: replace `"Obtained {!r} version for NimOS SDK package".`
 with `"Obtained {!r} version for Nimble Python SDK package`

**Pull request merged**